test-kuttl/e2e/multi-sync-snapshot-rclone/10-sync-from-source.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: volsync.backube/v1alpha1
+kind: ReplicationSource
+metadata:
+  name: source
+spec:
+  sourcePVC: data-source
+  trigger:
+    manual: firstsync
+  rclone:
+    rcloneConfigSection: "rclone-data-mover"
+    rcloneDestPath: "volsync-test-bucket2"
```

Comment:
We should get in the habit of naming the minio destinations based on the test so we don't end up w/ collisions across tests.

Reply:
Thanks for fixing `simple-rclone` also!